src/components/AccountSelector/AccountSelectButton/AccountSelectorButton.tsx

```diff
@@ -0,0 +1,56 @@
+import { ComponentPropsWithoutRef } from "react";
+
+export interface AccountSelectorProps extends ComponentPropsWithoutRef<"div"> {
+  theme?: string;
+  profileimage: React.ReactNode;
+  accountname: string;
+  svg: React.ReactNode;
+  accountbalance: string;
+  children?: React.ReactNode;
+}
+
+export function AccountSelectorButton(props: AccountSelectorProps) {
+  // Specific classes for the account selector button
+  const accountSelectorClass = "flex flex-row items-center justify-between";
+
+  const hoverClass = `hover:bg-info`;
+  const activeClass = `active:bg-c-pressed`;
+
+  // Generic classes
+  const genericFlex = "flex justify-center items-center";
+
+  const tokenDimensionClass = `h-6 w-6`;
```

Comment:
```suggestion
```
Remove this constant because it is already present at line 46.

Reply:
done !